apps/framework-docs/src/pages/aurora/reference/tool-reference.mdx

```diff
@@ -6,152 +6,600 @@ description: MCP tools and external tools for Aurora
+- *"Build a real-time analytics system for aircraft tracking data with location-based queries"*
+- *"Create a data pipeline that processes IoT sensor data and provides aggregated insights via REST APIs"*
+
+**Tools used:** `write_spec`, `write_ingestion_pipeline`, `write_workflow`, `write_stream_function`, `write_materialized_view`, `create_egress_api`
+
+</ToggleBlock>
+
+<ToggleBlock openText="Data Product Development" closeText="Data Product Development">
+
+When you want to create optimized data products for specific use cases.
+
+**Sample prompts:**
+- *"Create a data product for visualizing user engagement metrics with real-time updates"*
+- *"Build materialized views that improve query performance for my analytics dashboard"*
+- *"Create consumption APIs that support my frontend application's data needs"*
+
+**Tools used:** `write_materialized_view`, `create_egress_api`, `test_egress_api`
+
+</ToggleBlock>
+
+<ToggleBlock openText="Data Quality & Troubleshooting" closeText="Data Quality & Troubleshooting">
+
+When you need to fix data issues or improve data processing reliability.
+
+**Sample prompts:**
+- *"Look at my DLQ topic and create a streaming function to fix the data format issues"*
+- *"Create a data validation workflow that checks data quality before processing"*
+- *"Build error handling for my data ingestion pipeline"*
+
+**Tools used:** `read_redpanda_topic`, `write_stream_function`, `write_workflow`
+
+</ToggleBlock>
+
+<ToggleBlock openText="Performance Optimization" closeText="Performance Optimization">
 
-If you are managing the Aurora MCP with `mcp.json`, you can enable this toolset by adding the following to your `mcp.json`:
+When you want to improve the speed and efficiency of your data products.
 
-```json filename="MCP.json" copy
-"args": [
-    ...
-  "--moose-tools", 
-  ...
-]
+**Sample prompts:**
+- *"Create materialized views that will improve the performance of my user analytics APIs"*
+- *"Analyze query patterns and optimize my data model for better performance"*
+- *"Create pre-computed aggregations for my most common dashboard queries"*
+
+**Tools used:** `write_materialized_view`, `create_egress_api`, `test_egress_api`
+
+</ToggleBlock>
+
+### Tools
+
+#### `write_spec`
+Generates a plan for an end to end Moose project (from ingest, through transformation and egress). 
+
+Files created:
+* *Creates a .md file in the current directory with the plan.*
+
+<ToggleBlock openText="Show sample specification output" closeText="Hide sample specification output">
+
+```txt filename="/project/app/specs/specification.md" copy
+# Technical Specification: ASD-B Aircraft Data Ingestion and Visualization
+
+## 1. Overview
+
+This specification outlines the implementation of a system to ...
+```
+
+</ToggleBlock>
+
+#### `write_and_run_temp_script`
+Creates and runs a temporary script, usually for sampling purposes.
+
```

Comment:
also great for API inspection - which is sampling - but might be good to overly call it out.